libevm/VMCalls.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+#include "VM.h"
+#include <libethereum/ExtVM.h>
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+
+
+
+void VM::copyDataToMemory(bytesConstRef _data, u256*& SP)
+{
+	auto offset = static_cast<size_t>(*SP--);
+	s512 bigIndex = *SP--;
+	auto index = static_cast<size_t>(bigIndex);
```

Comment:
Not any more.  I still need to review the code for every remaining place with a too-wide type, and for uint64_t values passed to functions that take a size_t.  (You changed some size_t a while back.)